examples/static-react/client/src/App.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from 'react';
+import './App.css';
+
+// Clock documentation: https://projects.wojtekmaj.pl/react-clock/
+// and https://github.com/wojtekmaj/react-clock
+import Clock from 'react-clock';
+import 'react-clock/dist/Clock.css';
+
+function App() {
+  const [value, setValue] = useState(new Date());
```

Comment:
@rogertheriault - What do you think to rename the state to something like `date`, `time` or `timeValue`? The docs from react-clock have `value` as an example, but I think it's too generic.

Reply:
Yes, I was thinking of that, I was torn between leaving the example as is and correcting that one thing for readability. I think I'll rename it to `clocktime`.